### PR TITLE
Clarify sentence in CONTRIBUTING

### PR DESCRIPTION
**Pull request by lishaduck**
There was a missing comma, so I fixed it and updated how the sentence sounded to be a bit more natural.

## Description

n/a

## Checklist:

n/a

### CODE REVIEW

**Review by lildude — APPROVED** [review]

**Comment by Lhw95lev** [comment]
LHW95HAK #lhw 🥀 #LHW95HAK 3
24
0795959898
Day 24
Email lhwlev@gmail.com

**Review by Garfield-FatOrangeKat — COMMENTED** [review]
Disregard

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ If you can, try to reproduce the highlighting problem in the text editor that th
 You can also try to fix the bug yourself and submit a pull-request.
 [TextMate's documentation](https://manual.macromates.com/en/language_grammars) offers a good introduction on how to work with TextMate-compatible grammars.
 Note that Linguist uses [PCRE](https://www.pcre.org/) regular expressions, while TextMate uses [Oniguruma](https://github.com/kkos/oniguruma).
-Although they are mostly compatible there might be some differences in syntax and semantics between the two.
+Although they are mostly compatible, there may be occasional differences in syntax and semantics between the two.
```
Comment (Garfield-FatOrangeKat) [inline CONTRIBUTING.md:169]: I AM IN CONTROL NOW

**Review by Garfield-FatOrangeKat — COMMENTED** [review]
Attic

**Comment by Garfield-FatOrangeKat** [comment]
> Disregard 

[[[[]()]()]()]()

**Review by Garfield-FatOrangeKat — COMMENTED** [review]
- [ ] I am the owner

**Comment by Garfield-FatOrangeKat** [comment]
> - [ ] I am the owner

**Pull request merged**